feature_layers/display-clusters/src/main/java/com/esri/samples/display_clusters/DisplayClustersSample.java

```diff
@@ -0,0 +1,171 @@
+ * the License.
+ */
+
+package com.esri.samples.display_clusters;
+
+import java.util.concurrent.ExecutionException;
+
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.popup.Popup;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.IdentifyLayerResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import javafx.application.Application;
+import javafx.geometry.Point2D;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.layout.VBox;
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+import javafx.scene.control.Button;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+
+public class DisplayClustersSample extends Application {
+
+  private ArcGISMap map;
+  private MapView mapView;
+  private PortalItem portalItem; // keep loadable in scope to avoid garbage collection
+  private FeatureLayer powerPlantsLayer;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Points Using Clustering Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a button to toggle clustering
+      Button featureClusteringToggle = new Button("Toggle Feature Clustering");
+      featureClusteringToggle.setDisable(true);
+
+      // create a portal and portal item, using the portal and item ID
+      var portal = new Portal("https://www.arcgis.com/");
+      portalItem = new PortalItem(portal, "8916d50c44c746c1aafae001552bad23");
+
+      map = new ArcGISMap(portalItem);
+      map.addDoneLoadingListener(() -> {
+        if (map.getLoadStatus() == LoadStatus.LOADED) {
+          featureClusteringToggle.setDisable(false);
+
+          powerPlantsLayer = (FeatureLayer)map.getOperationalLayers().get(0);
+
+          // when the button is clicked, toggle the feature clustering
+          mapView.setOnMouseClicked(mouseEvent -> {
+            if (mapView == null) {
+              return;
+            }
+
+            mapView.getCallout().setVisible(false);
```

Comment:
Our [Callout API docs](https://developers.arcgis.com/java/api-reference/reference/com.esri.arcgisruntime/com/esri/arcgisruntime/mapping/view/Callout.html) say:
Call...  [dismiss()](https://developers.arcgis.com/java/api-reference/reference/com.esri.arcgisruntime/com/esri/arcgisruntime/mapping/view/Callout.html#dismiss()) to hide it (this will clean up any internal resources, so prefer calling it over changing the visibility directly.)